gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageReadChannel.java

```diff
@@ -929,7 +946,19 @@ protected InputStream openStream(long bytesToRead) throws IOException {
     Get getObject = createDataRequest(rangeHeader);
     HttpResponse response;
     try {
+      startTimeOfChannelOpen = System.currentTimeMillis();
       response = getObject.executeMedia();
+      long requestDelay = System.currentTimeMillis() - startTimeOfChannelOpen;
+      logger.atFine().log(
+          "openStream complete context:%d,time:%d,bytesToRead:%d,rangeSize:%d,resource:%s,requestId:%s",
+          Thread.currentThread().getId(),
+          requestDelay,
+          bytesToRead,
+          contentChannelEnd - contentChannelPosition,
+          resourceId,
+          requestDelay > 1000
```

Comment:
We should maybe log all the time because 1000 is arbritrary constraint and we could do analysis of these logs based on "time:" field in log? Also 1000 should be extracted as a const.